test/fulfillment_order_test.rb

```diff
@@ -49,5 +48,83 @@ def setup
         assert_equal 450789469, fulfillment.order_id
       end
     end
+
+    context "#move" do
+      should "be able to move fulfillment order to a new_location_id" do
+        fulfillment_order = ShopifyAPI::FulfillmentOrder.find(519788021)
+        new_location_id = 5
+
+        original = fulfillment_order.clone
+        original.status = 'closed'
+        moved = ActiveSupport::JSON.decode(load_fixture('fulfillment_order'))
+        moved['assigned_location_id'] = new_location_id
+
+        body = {
+            original_fulfillment_order: original,
+            moved_fulfillment_order: moved,
+            remaining_fulfillment_order: nil,
+        }
+        fake_query = "?fulfillment_order%5Bnew_location_id%5D=5"
```

Comment:
Does ShopifyAPI add these url params when making the request to Shopify? For a POST, shouldn't this be in the payload?

Reply:
I was wondering this exact thing when I first noticed it. I've been tophatting these changes with actual requests to the REST api and they seem to work. I suspect that the back-end is reading params from either the body or url. I can investigate further to make the tests work with POSTed bodies.

Reply:
Now making requests with posted content rather than query params.